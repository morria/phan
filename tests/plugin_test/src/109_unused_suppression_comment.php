<?php
namespace NS650;
/**
 * @var int
 * @phan-suppress PhanUnextractableAnnotation suppress should work.
 */
function example(): string {
    return 'str';
}
echo example();

/**
 * @var int
 * @phan-suppress PhanParamTooMany
 * (UnusedSuppressionPlugin should warn about PhanParamTooMany)
 */
function example2(): string {
    return 'str';
}
echo example2();
