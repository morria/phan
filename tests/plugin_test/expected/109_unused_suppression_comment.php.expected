src/109_unused_suppression_comment.php:13 PhanUnextractableAnnotation Saw unextractable annotation for comment '* @var int'
src/109_unused_suppression_comment.php:17 UnusedSuppression Element \NS650\example2 suppresses issue PhanParamTooMany but does not use it
